packages/react/src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -43,36 +43,36 @@ export const ProgressStep = ({ ...props }) => {
     }
   };
 
-  const currentSvg = current && (
-    <svg>
-      <path d="M 7, 7 m -7, 0 a 7,7 0 1,0 14,0 a 7,7 0 1,0 -14,0" />
-      <title>{description}</title>
-    </svg>
-  );
-
-  const completeSvg = complete && (
-    <CheckmarkOutline16 aria-label={description} role="img">
-      <title>{description}</title>
-    </CheckmarkOutline16>
-  );
-  const incompleteSvg = (() => {
-    if (complete) {
-      return null;
-    }
+  const SVGIcon = () => {
```

Comment:
Given you made this React component, would you want to remove closure ref dependency? e.g. Making `invalid`/`description`, etc. React props for this component.